src/commands/manifest.ts

```diff
@@ -99,16 +113,30 @@ export default class Manifest extends Command {
     this.log(`wrote manifest to ${file}`)
   }
 
-  private cloneRepo(repoUrl: string, fullPath: string, tag: string | semver.SemVer | null): void {
-    try {
-      this.executeCommand(`git clone --branch ${tag} ${repoUrl} ${fullPath} --depth 1`)
-    } catch {
-      try {
-        this.executeCommand(`git clone --branch v${tag} ${repoUrl} ${fullPath} --depth 1`)
-      } catch {
-        throw new Error(`Unable to clone repo ${repoUrl} with tag ${tag}`)
+  private getTarballUrl(plugin: string, version: string): string {
+    // jit plugin is unpinned so we need to figure out the max satisfying version
+    if ((version.startsWith('^') || version.startsWith('~'))) {
+      const npmLatest = JSON.parse(this.executeCommand(`npm view ${plugin}@latest --json`).stdout) as {
```

Comment:
`@latest` has no effect on the output and can be removed.  It's also confusing because we don't want latest, we want everything (ex: the jit version is `^4.0.0-beta.0`)

Reply:
`@latest` is in there because of an interesting npm feature we discovered while trying to release the JIT feature. Here's the PR that introduced it: https://github.com/oclif/oclif/pull/1107

`oclif manifest` gets run during `prepack` which means `npm` has set `npm_config_tag` to the requested tag (could be `nightly`, `latest-rc`, or `latest`), which is used for any `npm` command. So if we don't explicitly set `latest` then it will try to request a tag that doesn't exist for our plugins

If you request `latest`, you'll still get all the versions - even the ones that are ahead of latest. For instance, `npm view @oclif/core@latest versions` shows all the v3 beta versions even though latest is currently 2.15.0